curvefs/src/client/warmup/warmup_manager.h

```diff
@@ -206,6 +227,8 @@ class WarmupManager {
         kvClientManager_ = std::move(kvClientManager);
     }
 
+    using Filepath2WarmupProgressMap = std::unordered_map<std::string,
```

Comment:
Use inodeid to manage warmup tasks instead of paths

Reply:
It is a data structure solely for the output.

e.g., 
path1: 20/123
path2: 241/43881
...

Reply:
> This data structure serves only for output purposes and isn't intended to replace the current `inode2Progress_`. e.g., path1: 20/123 path2: 241/43881 ...

If you want to output inodeid, you can feel it. If you want to output path, you can do it through tools. Otherwise, pay attention to recycling invalid data in the data structure.